app/views/static/digital_scholarship.html.erb

```diff
@@ -0,0 +1,734 @@
+                                    </div>
+                                </div> 
+                            </div>
+                        </div>
+
+             
+                        
+                    </div>
+                    
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- Crowd Case End -->
+
+    <!-- Crowd Testimonials -->
+    <section class="crowd-testimonials">
+        <div class="container">
+            <div class="row justify-content-center">
+                <div class="col-lg-9">
+                    <div class="third-title">
+                        <h4>What Others Are Saying About FromThePage</h4>
+                        <!-- <p>For both research and pedagogy; not having to wrestle with a tool is so helpful in keeping up the momentum of the work</p> -->
+                    </div>
+
+                    <div class="loop">
+
+                        <div class="box">
+                            <div class="item">
+                                <p><img src="assets/icons/quote01.svg" alt="" class="top">I have used FromThePage as both a research and a pedagogical tool; not having to wrestle with the tool is so helpful in keeping up the momentum of the work. I can confidently ask my colleagues to collaborate without asking them to spend hours learning a new tool or interface. A four year remote collaboration project has endured to a great extent because we could bring project members in seamlessly. In the classroom, my students have found transcribing exciting and gratifying, and a wonderful addition to the secondary material we are learning. Students really like learning this way because it is an unmediated experience; this is a real experience of meeting a scribe through his or her writings.<img src="assets/icons/quote02.svg" alt="" class="bottom"></p>
+
+                                <div class="info">
+
+                                    <div class="text">
+                                        <h4>Dr. Laura Morreale</h4>
+                                        <span>Scholar of Medieval Francophonie and Italian Historiographies</span>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="box">
+                            <div class="item">
+                                <p><img src="assets/icons/quote01.svg" alt="" class="top">We have an enormous amount of understudied documentation that could yield meaningful findings, if people could just get to the stuff — FromThePage helps make access possible.
+<br>
```

Comment:
Indent this (and similar second/third lines) under the `p`

Reply:
fixed.